utils/message_html_to_component.jsx

```diff
@@ -60,6 +60,11 @@ export function messageHtmlToComponent(html, isRHS, options = {}) {
                     <LinkTooltip
                         href={node.attribs[hrefAttrib]}
                         title={children[0]}
+                        dataAttributes={{
+                            'data-hashtag': attribs['data-hashtag'],
+                            'data-link': attribs['data-link'],
+                            'data-channel-mention': attribs['data-channel-mention'],
```

Comment:
Alternatively, we can pass all values in `attribs`, but they would not be used by `LinkTooltip`.